README.md

```diff
@@ -1,24 +1,167 @@
-# README
+# Budget-App
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+<a name="readme-top"></a>
 
-Things you may want to cover:
+# 📗 Table of Contents
 
-* Ruby version
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
 
-* System dependencies
+<!-- PROJECT DESCRIPTION -->
 
-* Configuration
+# 📖 [Budget-App] <a name="about-project"></a>
 
-* Database creation
+**[Budget-App]** The Budget App is about building a mobile web application where users can manage their budget: They have a list of transactions associated with a category, so that users can see how much money they spent and on what.
 
-* Database initialization
+## 🛠 Built With <a name="built-with"></a>
 
-* How to run the test suite
+### Tech Stack <a name="tech-stack"></a>
 
-* Services (job queues, cache servers, search engines, etc.)
+<details>
+  <summary>Ruby</summary>
+  <ul>
+<li>
 
-* Deployment instructions
+<a name="livelink"></a>
 
-* ...
+- [Click here to read more about ruby](https://www.ruby-lang.org/en/)
+
+   </li>
+  </ul>
+
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[Add groups]**
+- **[Add Entries]**
+- **[Display groups and entries]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+```sh
+ bundle install
+```
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd Budget
+  git clone https://github.com/MayPyone/Budget-App.git
+```
+
+### Install
+
+Install this project with:
+
+```sh
+  cd Budget
+  bundle install
+  gem install --no-document rubocop -v '>= 1.0, < 2.0'
+  gem install rspec
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  ruby main.rb
+  or
+  ./main
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+- `rubocop --color`
+- `rspec spec --format doc`
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Deployment
+
+-
+
```

Comment:
- [ Required ] Good job working on this project, however, I think it would be nice if you included a link to a video presentation as stated in the project requirements. A presentation video serves to inform someone interested in your project such as a recruiter about the features you've worked on. Please see the screenshot below for all the items you need to include while making the video presentation.

## Project requirements for the video presentation
![Screenshot from 2023-12-17 00-13-48](https://github.com/MayPyone/Budget-App/assets/16392046/2a0886c4-52db-412d-b19d-935d158afa2d)